src/death/corpses.ts

```diff
@@ -22,43 +22,36 @@ const BLOOD_MATERIAL = Material.DEAD_BUBBLE_CORAL_FAN;
 
 const Body = new CustomItem({
   id: 1,
-  modelId: 1,
   type: HIDDEN_MATERIAL,
```

Comment:
Kinda unrelated, but note to self that HUDDEN_MATERIAL could be replaced with VkItem.